File: countDigit.py
num = int(input("Enter a number: ")) 
target_digit = int(input("Enter the target digit: ")) 
count = 0 
while num>0:
    digit=num%10
    if digit== target_digit:
        count+=1
    num//=10
print("Count of digit", target_digit, "is:", count)

File: countDigit.java
import java.util.Scanner;

public class countDigit {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number: "); 
  int num=sc.nextInt(); 
  System.out.println("Enter the target Numbder: "); 
  int target=sc.nextInt();
  int count=0; 
  for(int i=0;i<num;i++) { 
   int dig=num%10; 
   if(dig==target) { 
    count++; 
   } 
   num=num/10; 
} 
System.out.println(count);
  
    }
}
